meteor/client/ui/Prompter/PrompterView.tsx

```diff
@@ -792,15 +801,10 @@ export const Prompter = translateWithTracker<IPrompterProps, {}, IPrompterTracke
 				segment.parts.forEach((part) => {
 					lines.push(
 						<div
-							id={`part_${part.id}`}
+							id={`partInstance_${part.id}`}
```

Comment:
Minor note: should `part` be renamed to `partInstance` as well?

Reply:
The whole Prompter API just needs reworking. It's using it's own set of PrompterDataSegment and PrompterDataPart that then borrows the source's Ids, so the naming is way too confusing. I don't want to spend too much time on this, we need to refactor the Prompter to actually have a sound architecture.